test/tests/no-root-password/container.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+
+[ "$(grep root /etc/shadow | cut -d: -f2)" = '*' ]
```

Comment:
I also wonder whether we can/should check against the hashed version of the empty string instead of just `*`.